sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/SchemaUtil.java

```diff
@@ -67,7 +67,7 @@
 
 /** Provides utility functions for working with Beam {@link Schema} types. */
 @Experimental(Kind.SCHEMAS)
-class SchemaUtil {
+public class SchemaUtil {
```

Comment:
module private class (no public modifier) should still be accessible in JdbcIO?

Reply:
I made it public so its accessible in the templates repository. I think the Dataplex Jdbc template needs this class.

Reply:
I think unit tests for this class already exist